capa/features/extractors/vmray/extractor.py

```diff
@@ -55,12 +55,16 @@ def extract_file_features(self) -> Iterator[Tuple[Feature, Address]]:
     def extract_global_features(self) -> Iterator[Tuple[Feature, Address]]:
         yield from self.global_features
 
+    def get_import_names(self) -> Iterator[Tuple[Feature, Address]]:
+        for filename_id, filename_data in self.analysis.filenames.items():
+            yield String(filename_data["filename"]), NO_ADDRESS
+
     def get_processes(self) -> Iterator[ProcessHandle]:
         yield from capa.features.extractors.vmray.file.get_processes(self.analysis)
 
     def extract_process_features(self, ph: ProcessHandle) -> Iterator[Tuple[Feature, Address]]:
         # TODO (meh): https://github.com/mandiant/capa/issues/2148
-        yield from []
+        yield from capa.features.extractors.vmray.file.extract_features(self.analysis)
```

Comment:
We shouldn't emit file features as process features here (see capa's [CAPE extractor implementation](https://github.com/mandiant/capa/blob/b617179525b86297e2541722e7a8832a49df59e2/capa/features/extractors/cape/extractor.py#L62-L63) as a guide). However, capa's CAPE extractor presently only emits process environment strings as process features and I don't believe VMRay gives us that information so we should revert this change for now and focus on finalizing the rest of this PR.